src/XrdTpc/XrdTpcTPC.cc

```diff
@@ -442,28 +439,32 @@ int TPCHandler::DetermineXferSize(CURL *curl, XrdHttpExtReq &req, State &state,
                                   bool &success, TPCLogRecord &rec, bool shouldReturnErrorToClient) {
     success = false;
     curl_easy_setopt(curl, CURLOPT_NOBODY, 1);
+    // Set a custom timeout of 120 seconds (= CONNECT_TIMEOUT for convenience) for the HEAD request
```

Comment:
Hmmm, as far as I can see CONNECT_TIMEOUT is defined as 60 so why is the comment saying 120?

Reply:
Yes, typo from me, will fix.